app/main.py

```diff
@@ -1,6 +1,31 @@
+import os
+import requests
+
+API_KEY = os.environ["API_KEY"]
+
+CITY = os.environ.get("CITY", "Paris")
+URL = (
+    f"http://api.openweathermap.org/data/2.5/weather"
+    f"?q={CITY}"
+    f"&appid={API_KEY}"
+    f"&units=metric"
+)
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    try:
+        response = requests.get(URL)
+        response.raise_for_status()
+
+        data = response.json()
+        temperature = data["main"]["temp"]
+        weather_description = data["weather"][0]["description"]
+
+        print(f"Temperature in Paris: {temperature}°C")
```

Comment:
Bug: The city name in the output message is hardcoded as 'Paris'. It should be replaced with the value of the CITY variable to reflect the actual city for which the weather information is fetched.